src/Application/Actions/Hooks/StripeUpdate.php

```diff
@@ -66,6 +67,9 @@ protected function action(): Response
                 case 'charge.succeeded':
                     $this->handleChargeSucceeded($event);
                     break;
+                case 'payout.paid':
+                    $this->handlePayoutPaid($event);
```

Comment:
Unit tests?!

Reply:
@NoelLH I was going to ask on Friday, how do I mock a Stripe API response?

Reply:
@tbg-jasonfung it should be very similar to all the other places we use Prophecy. You need to prophesise `StripeClient`. This is another reason to use the injected service instead of creating a new copy in the action.